### ISSUE TITLE
SQLFlow doesn't recognize an optimizer name

### ISSUE THREAD

**Issue by shendiaomo** [sql-machine-learning/sqlflow#1275]
**Description**
Input:
```sql
SELECT * FROM iris.train TO TRAIN DNNClassifier WITH model.hidden_units=[300,200], model.n_classes=3, model.optimizer="RMSProp" LABEL class INTO sqlflow_models.test_dnn_model;
```
Output:
```
==========Output==========
Traceback (most recent call last):
  File "<stdin>", line 68, in <module>
NameError: name 'RMSProp' is not defined
```

**Comment by typhoonzero** [comment]
Is it a typo `model.optimizer`? https://www.tensorflow.org/api_docs/python/tf/estimator/DNNClassifier#__init__

**Comment by shendiaomo** [comment]
> Is it a typo `model.optimizer`? https://www.tensorflow.org/api_docs/python/tf/estimator/DNNClassifier#__init__

Sorry but it's just a typo in the issue, the actual SQL statement is right. It doesn't work because the generated code didn't quote `RMSProp`. I fix this in #1278 
![image](https://user-images.githubusercontent.com/4180295/69820718-ba3b2f00-123c-11ea-985d-afa413d5fedb.png)

**Comment by shendiaomo** [comment]
This is fixed.
